lms/djangoapps/badges/tests/test_models.py

```diff
@@ -0,0 +1,247 @@
+"""
+Tests for the Badges app models.
+"""
+from django.core.exceptions import ValidationError
+from django.core.files.images import ImageFile
+from django.test import TestCase
+from django.test.utils import override_settings
+from mock import patch, Mock
+from nose.plugins.attrib import attr
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from badges.models import CourseCompleteImageConfiguration, validate_badge_image, BadgeClass, BadgeAssertion
+from badges.tests.factories import BadgeClassFactory, BadgeAssertionFactory, RandomBadgeClassFactory
+from certificates.tests.test_models import TEST_DATA_ROOT
+from student.tests.factories import UserFactory
+
+
+class ImageFetchingMixin(object):
```

Comment:
I mentioned this in another comment: there's no reason for this to be a mixin, or a class.  It's a one-line helper function.